packages/frontend/src/app/routing/app-routing.module.ts

```diff
@@ -22,14 +22,14 @@ import {BuildComponent} from '../build/build.component';
 import {NotFoundComponent} from '../not-found/not-found.component';
 import {apiLinks} from '../services/com/api';
 import {LoginGuard} from './route-guards/LoginGuard';
+import {RouteProvider} from './route-provider/RouteProvider';
 
 const routes: Routes = [
   {path: '', component: HomeComponent},
   {path: 'license', component: LicenseComponent},
-  {path: 'search', component: MainComponent},
```

Comment:
As the organisation name will have to be known by the user, the search route isn't relevant.